software/bootcamp-code/main.py

```diff
@@ -33,61 +107,196 @@ def main_loop():
     # open the serial connection
     omni_motion.open()
     
+    # list for referee commands
+    command_list=[]
+    if referee_active:
+        # listen for referee commands if the referee is active
+        asyncio.get_event_loop().run_until_complete(listen_referee(command_list))
```

Comment:
What happens if the referee server resets or there is a connection issue?

Reply:
Should adding ayncio.get_event_loop().run_until_complete(listen_referee(command_list)) to the if statement fix that?